styles.css

```diff
@@ -24,12 +23,12 @@ a:visited {
 
 a:hover {
```

Comment:
We need to respect the design regarding the state of the buttons too.

- [ ] Please adapt your layout for buttons in order to respect the design in accordance with the state it has as we have them in the Figma design: :+1: 
![image](https://user-images.githubusercontent.com/83596135/160186692-26a4b40f-9006-4716-838d-b6d20f13ae40.png)